content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
+
+The goal of cycle counts is to count more often at key locations to keep your critical stock levels
```

Comment:
- Restructure sentence so it's less awkward
- Replace the sentence on Lines 9-10 `The goal of...accurate.` with `The goal of cycle counts is to keep critical stock levels accurate by counting more often at key locations.`